src/renderer/components/interact/forms/Unbond.tsx

```diff
@@ -30,22 +33,25 @@ export const Unbond: React.FC<Props> = ({ onFinish: onFinishProp, isLoading = fa
     [onFinishProp]
   )
 
+  // graterThan returns pure function and there is no need to calidate its deps
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const graterThenZero = useCallback(
+    greaterThan(ZERO_BN)(intl.formatMessage({ id: 'wallet.validations.graterThen' }, { value: 0 })),
+    [intl]
+  )
```

Comment:
Same here: Just use it directly in `amountValidator`, so that we have just one validation function with `amountValidator`.